consensus/scheme/rolldpos/endorsedconsensusmessage.go

```diff
@@ -89,7 +92,8 @@ func (ecm *EndorsedConsensusMessage) LoadProto(msg *iotextypes.ConsensusMessage)
 		ecm.message = vote
 	case msg.GetBlockProposal() != nil:
 		proposal := &blockProposal{}
-		if err := proposal.LoadProto(msg.GetBlockProposal()); err != nil {
+		pb := msg.GetBlockProposal()
+		if err := proposal.LoadProto(pb, ecm.checker(pb.Block.Header.Core.Height)); err != nil {
```

Comment:
@CoderZhi  or should use `msg.Height` here? is `msg.Height` equal to `pb.Block.Header.Core.Height`?

Reply:
they should be the same